### ISSUE TITLE
Publish to NPM

### ISSUE THREAD

**Issue by benjick** [acupofspirt/rust-experimental-webrcon#1]
Hey,

Is this published?

**Comment by acupofspirt** [comment]
Nope, is it necessary? I can if nedeed, just let me know and I will publish.

**Comment by benjick** [comment]
It is not necessary, just asking so I don't check your package in to my repo if not needed. Thanks!

Sidequestion, is it possible to get the console streamed? I'm curious how RustAdmin does it

**Comment by acupofspirt** [comment]
What do you mean by streamed?
